### ISSUE TITLE
Gadgets?

### ISSUE THREAD

**Issue by EpicBossCraft** [TotalFreedom/TotalFreedomMod#1496]
I have this idea to add gadgets to the TF Mod so players can play with gadgets

**Comment by DUFCLiam** [comment]
Cool, Like a jumper tool? Makes you orbit for 1s then stops

**Comment by DUFCLiam** [comment]
(Silently orbits)

**Comment by Adaptified** [comment]
I like this, but let's keep it minimal. Nothing too big or extravagant.

**Comment by JeromSar** [comment]
@CMackiee What kind of gadgets?

**Comment by Adaptified** [comment]
@JeromSar Maybe stuff like double jump, bow teleporter, etc.

**Comment by CMackiee** [comment]
And paint gun and bat blaster

**Comment by PieGuy7896** [comment]
@CMackiee - cough - mineplex - cough -

**Comment by Wild1145** [comment]
So we're looking to add some cool little extra's that players can use basically?

**Comment by JeromSar** [comment]
Yeah, I think something along these lines would be cool. However, the specific gadgets to be implemented will need to be specified in this issue.

**Comment by Wild1145** [comment]
I will look at splitting this down then today and figuring out exactly what "type" of gadgets could be interesting to implement.
